### PR TITLE
Utilize emulators:exec better for web frameworks integration test

### PR DESCRIPTION
**Pull request by jamesdaniels**

### CODE REVIEW

File: scripts/webframeworks-deploy-tests/tests.ts [hunk 1]
```diff
@@ -29,44 +18,27 @@ async function getFilesListFromDir(dir: string): Promise<string[]> {
...
   });
 
   after(() => {
     // This is not an empty block.
   });
 
-  it("should log reasons for backend", () => {
+  it("should log reasons for backend", async () => {
     process.env.FIREBASE_CLI_EXPERIMENTS = "webframeworks";
+    const result = (await readFile("firebase-emulators.log")).toString();
```
Comment (jamesdaniels) [inline scripts/webframeworks-deploy-tests/tests.ts:33]: can switch to sync, async doesn't get us anything IMO

File: .gitignore [hunk 2]
```diff
@@ -22,3 +22,4 @@ scripts/*.json
 
 lib/
 dev/
+clean/
```
Comment (jamesdaniels) [inline .gitignore:25]: newline

File: scripts/webframeworks-deploy-tests/tests.ts [hunk 3]
```diff
@@ -59,14 +30,17 @@ describe("webframeworks deploy build", function (this) {
 
   it("should log reasons for backend", () => {
     process.env.FIREBASE_CLI_EXPERIMENTS = "webframeworks";
```
Comment (leoortizz) [inline scripts/webframeworks-deploy-tests/tests.ts:32]: do we still need this?

Comment (jamesdaniels) [reply scripts/webframeworks-deploy-tests/tests.ts:32]: nope, we can cut

**Review by leoortizz — APPROVED** [review]
I think we can [remove this](https://github.com/firebase/firebase-tools/pull/5781/files#diff-25ac90f12cd66fc5151f140c9b37c506918d3518ed652df066a06088065c3150L61) and merge :)

**Pull request merged**